packages/react/src/QuestionnaireBuilder/QuestionnaireBuilder.tsx

```diff
@@ -406,12 +423,15 @@ function ItemBuilder<T extends Questionnaire | QuestionnaireItem>(props: ItemBui
               href="#"
               onClick={(e: React.MouseEvent) => {
                 e.preventDefault();
-                addItem({
-                  id: generateId(),
-                  linkId: generateLinkId('g'),
-                  type: 'group',
-                  text: 'Group',
-                } as QuestionnaireItem);
+                addItem(
+                  {
+                    id: generateId(),
+                    linkId: generateLinkId('g'),
+                    type: 'group',
+                    text: 'Group',
+                  } as QuestionnaireItem,
+                  true
```

Comment:
For adding Groups and Pages, there is an error for no nested items. I disabled the AutoSubmit for these two specifically
<img width="503" alt="Screenshot 2023-11-06 at 4 12 53 PM" src="https://github.com/medplum/medplum/assets/6913745/3379e1ad-1d7f-4317-968b-cd363e612792">